sale_order_invoicing_queued/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+This module allows to enqueue in several jobs the sales orders invoicing
+process to be executed in paralell on background, that's it's normally done
```

Comment:
```suggestion
process to be executed in paralell on background, that is usually done
```

Reply:
OK about `'s`, but I have chosen normally because the etymological difference about what is "the rule": https://www.google.com/search?client=ubuntu&channel=fs&q=normally+vs+usually&ie=utf-8&oe=utf-8